includes/definitions.inc.php

```diff
@@ -1058,7 +1058,7 @@
 
 // EMC FlareOS
 $os = 'flareos';
-$config['os'][$os]['text'] = 'EMC CLARiiON';
+$config['os'][$os]['text'] = 'FLARE OS';
```

Comment:
as far as I know the official name is FLARE OS (Fibre Logic Array Runtime Environment OS).